docs/en/operations/server-configuration-parameters/settings.md

```diff
@@ -1553,6 +1639,82 @@ For more details, see [GraphiteMergeTree](../../engines/table-engines/mergetree-
 </graphite_rollup_example>
 ```
 
+## google_protos_path
+
+Defines a directory containing proto files for Protobuf types.
+
+By default, it is configured as:
```

Comment:
`programs/server/config.xml` only happens to configure the setting to `/usr/share/clickhouse/protos/`. If setting `<google_protos_path>` was not specified, the setting is empty by default. We should change l. 1646 to "Example:".